### PR TITLE
Controllers should dispatch events

### PR DESCRIPTION
**Pull request by carlescliment**
I've modified controllers to dispatch events after ipn handling, so that other bundles can make their business properly.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
```
Comment (carlescliment) [inline composer.json:0]: Trying to define an vcs instead of a separate repo in packagist. No luck :(

**Comment by alexanderdean** [comment]
Hi, thanks for this. Can you re-submit without the changes to composer.json? The other changes look fine.

**Comment by carlescliment** [comment]
Hi, you can dismiss this pull request. 

I've added a more cleaner one.

Thanks!

**Comment by alexanderdean** [comment]
Thanks Carles! Closing

**Pull request closed without merging**